src/latexify/test_stubs.py

```diff
@@ -0,0 +1,4 @@
+"""Stubbed methods for tests."""
```

Comment:
Technically, we don't need this kind of file, because:

- Every function is evaluated only when it is invoked
- latexify does not evaluate any inner stuff of the given function: we only "evaluates" the source.

Reply:
I'll remove it for now since we have `math.hypot`, but I only added it because `pflake8` complains about it otherwise. It doesn't cause any runtime errors though, just linting ones.